src/modules/histogramCN/histogramCN.cpp

```diff
@@ -18,7 +18,8 @@ HistogramCNModule::HistogramCNModule() : Module(ModuleTypes::HistogramCN)
     keywords_.addTarget<ConfigurationKeyword>("Configuration", "Set target configuration for the module", targetConfiguration_);
 
     keywords_.setOrganisation("Options", "Sites", "Specify sites defining the interaction A...B.");
-    keywords_.add<SpeciesSiteVectorKeyword>("SiteA", "Set the site(s) 'A' which are to represent the reference origin", a_);
+    keywords_.add<SpeciesSiteVectorKeyword>("SiteA", "Set the site(s) 'A' which are to represent the reference origin", a_,
+                                            true);
```

Comment:
Don't need axes for this one.
```suggestion
    keywords_.add<SpeciesSiteVectorKeyword>("SiteA", "Set the site(s) 'A' which are to represent the reference origin", a_);
```